dancer.js

```diff
@@ -411,7 +411,9 @@
       }
 
       this.audio.addEventListener( 'progress', function ( e ) {
-        if ( e.currentTarget.duration ) {
+        var isSeekable = e.currentTarget.seekable && e.currentTarget.seekable.length > 0;
```

Comment:
`e.currentTarget.seekable` could be a variable, since you're accessing it twice.